torch/_refs/__init__.py

```diff
@@ -3614,6 +3614,23 @@ def repeat(a: Tensor, *repeat_shape) -> Tensor:
     return permuted_result.reshape(target_shape)
 
 
+def _nd_to_1d(a: Tensor) -> Tensor:
+    torch._check(a.numel() > 0)
+    torch._check(a.ndim > 0)
+    torch._check(a.is_contiguous())
+    return torch.as_strided(a, [a.numel()], [1])
+
+
+def _1d_to_2d(a: Tensor, dim0, dim1) -> Tensor:
```

Comment:
Since this file has a lot of other functions in it, you need to be more specific about what exactly these functions do, or move them into view_reshape_helper, or just don't bother factoring them as helpers at all

Reply:
I agree.